core/eolearn/tests/test_core_tasks.py

```diff
@@ -425,6 +426,11 @@ def test_extract_bands(test_eopatch):
     patch = move_bands(test_eopatch)
     assert np.array_equal(patch.data["MOVED_BANDS"], patch.data["REFERENCE_SCENES"][..., bands])
 
+    patch.data["MOVED_BANDS"][0, 0, 0, 0] += 1.0
+    assert math.isclose(
+        patch.data["REFERENCE_SCENES"][0, 0, 0, bands[0]] + 1.0, patch.data["MOVED_BANDS"][0, 0, 0, 0], rel_tol=0.2
+    )
+
```

Comment:
use `pytest`s `approx` function, since it plays nicely with the asserts.

I'd also prefer to memorize the old value, change it to a new one, then check that it did change in the moved bands but not in the reference scenes (where it should match the old value).